docs/src/configuration/app/app-reference.md

```diff
@@ -539,32 +539,34 @@ if other hooks fail, the app is still deployed.
-| Name   | Type     | Required | Description |
-| ------ | -------- | -------- | ----------- |
-| `spec` | `string` | Yes      |  The [cron specification](https://en.wikipedia.org/wiki/Cron#CRON_expression). |
-| `cmd`  | `string` | Yes      |  The command that's run. It's run in [Dash](https://en.wikipedia.org/wiki/Almquist_shell). |
+Cron jobs are [logged](../../development/logs.md) at `/var/log/cron.log`. 
 
-Minimum time between cron jobs being triggered:
+The following table shows the properties for each job:
 
-| Plan                | Time      |
-|-------------------- | --------- |
-| Professional        | 5 minutes |
-| Elite or Enterprise | 1 minute  |
+| Name        | Type                                         | Required | Description |
+| ----------- | -------------------------------------------- | -------- | ----------- |
+| `spec`      | `string`                                     | Yes      | The [cron specification](https://en.wikipedia.org/wiki/Cron#CRON_expression). |
+| `commands`  | A [cron commands dictionary](#cron-commands) | Yes      | A definition of what commands to run when starting and stopping the cron job. |
 
-For each app container, only one cron job can run at a time.
-If a new job is triggered while another is running, the new job is paused until the other completes.
-To minimize conflicts, a random offset is applied to all triggers.
-The offset is a random number of seconds up to 5 minutes or the cron frequency, whichever is smaller.
-Crons are also paused while activities such as [backups](../../dedicated/overview/backups.md) are running.
-The crons are queued to run after the other activity finishes.
-
-If an application defines both a `web` instance and `worker` instances, cron jobs run only on the `web` instance.
+### Cron commands
 
-To run cron jobs in a timezone other than UTC, set the [timezone property](#top-level-properties).
+| Name               | Type      | Required | Description |
+| ------------------ | --------- | -------- | ----------- |
+| `start`            | `string`  | Yes      | The command that's run. It's run in [Dash](https://en.wikipedia.org/wiki/Almquist_shell). |
+| `stop`             | `string`  | No       | The command that's issued to give the cron command a chance to shutdown gracefully, such as to finish an active item in a list of tasks. Issued when a cron task is interrupted by a user through the CLI or management console. If not specified, a `SIGTERM` signal is sent to the process. |
+| `shutdown_timeout` | `integer` | No       | The number of seconds after which a `SIGKILL` signal is sent to the process to force terminate it. The default is `10` seconds. |
 
-Cron jobs are [logged](../../development/logs.md) at `/var/log/cron.log`.
+```yaml {location=".platform.app.yaml"}
+crons:
+    mycommand:
+        spec: '*/19 * * * *'
+        commands:
+            start: sleep 60 && echo sleep-60-finished && date
+            stop: killall sleep
+            shutdown_timeout: 18
```

Comment:
`shutdown_timeout: 18` should be at the same level as commands this currently give

```

E: Error parsing configuration files:
    - applications./.platform.app.yaml.crons.mycommand.commands: Unrecognized keys in mapping; Unknown key "shutdown_timeout"
```

maybe this line https://github.com/platformsh/platformsh-docs/pull/2141/files#diff-be4402d2999f0d3186dd097ade9d8ab1ccf001322ed82809b876f2693ba67293R559 should be moved to the other table?

Reply:
Thanks for the catch @spk!